src/parseExpr_staged.jl

```diff
@@ -3,7 +3,29 @@
 #  License, v. 2.0. If a copy of the MPL was not distributed with this
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#addToExpression(ex::Number, c::Number) = ex + c
+# `a in b` is a comparison after JuliaLang/julia#13078
+const in_is_compare = VERSION >= v"0.5.0-dev+901"
+
+function tryParseIdxSet(arg::Expr)
+    if arg.head === :(=) || (!in_is_compare && arg.head === :in)
+        @assert length(arg.args) == 2
+        return true, arg.args[1], arg.args[2]
+    elseif (in_is_compare && arg.head === :comparison &&
+            length(arg.args) == 3 && arg.args[2] === :in)
+        return true, arg.args[1], arg.args[3]
+    end
+    return false, nothing, nothing
+end
+
+function parseIdxSet(arg::Expr, msg="Invalid loop condition")
+    parse_done, idxvar, idxset = tryParseIdxSet(arg)
+    if parse_done
+        return idxvar, idxset
+    end
+    error(msg)
```

Comment:
That makes sense. I add a `msg` parameter because I saw many places that needs to be fixed. Turns out they are almost the same so I end up didn't have to supply different messages for different ones.